packages/seacas/applications/epu/EP_ExodusEntity.h

```diff
@@ -106,14 +97,14 @@ namespace Excn {
     size_t entity_count() const { return elementCount; }
 
     char                     elType[MAX_STR_LENGTH + 1]{};
-    std::string              name_;
+    std::string              name_{""};
```

Comment:
`std::string`'s default constructor initializes it to the empty string, so you don't need to do this here.